log-router/templates/daemonset.yaml

```diff
@@ -18,8 +18,10 @@ spec:
       labels:
         app: {{ template "fluentd-router.name" . }}
         release: {{ .Release.Name }}
-      {{- if .Values.podAnnotations }}
       annotations:
+        checksum/fluentd-extraenv: {{ toYaml .Values.fluentd.extraEnv | sha256sum }}
+        checksum/reloader-extraenv: {{ toYaml .Values.reloader.extraEnv | sha256sum }}
```

Comment:
It's a good idea to pipe this through `| quote`. It may happen that the sha sum is valid decimal literal and annotation values must be strings

Reply:
Thanks your suggests !

```sha256sum``` is defined in [here](https://github.com/Masterminds/sprig/blob/master/crypto.go#L30), and return type is ```string```. So, I think no problem without ```| quote```, cloud I set it?

FYI: https://github.com/helm/helm/blob/master/docs/chart_template_guide/functions_and_pipelines.md

Reply:
It would still be a problem if the string turns out to be 1234 - it still need to be quotes.

However, it seems unlikely to happen - and even if it does, a single byte change to the yaml will fix it :)

Let's merge it like this, thanks again for your PR!